### ISSUE TITLE
^v2.0.0-beta.1 does not match minimum-stability (Craft 5.0.5)

### ISSUE THREAD

**Issue by sotnf** [verbb/social-share#5]
### Describe the bug

I have tried to install this plug-in via copy/paste command from the plug-in store and trying to let composer look for an installable version. 

**Error 1** - DDEV Plugin Command
` Problem 1
    - verbb/social-share 2.0.0-beta.1 requires verbb/auth ^2.0.0-beta.1 -> found verbb/auth[2.0.0-beta.1, 2.0.0-beta.2] but it does not match your minimum-stability.`

**Error 2** - Wildcard Plugin Version
` Problem 1
    - verbb/social-share[1.0.0, ..., 1.0.5] require craftcms/cms ^4.0.0 -> found craftcms/cms[4.0.0, ..., 4.8.10] but it conflicts with your root composer.json require (^5.0.0).`

### Steps to reproduce

1. Run Craft via DDEV (see additional info for DDEV config) in WSL
2. Attempt installation via DDEV Composer command.
3. `ddev composer require "verbb/social-share:^2.0.0-beta.1" -w && ddev craft plugin/install social-share`
4. Receive Error 1
5. Attempt installation with Composer suggestion of --with-all-dependencies (same error)
6. Attempt installation with Composer suggestion of a wildcard plug-in version.
7. `ddev composer require verbb/social-share:* && php craft plugin/install social-share`
8. Receive Error 2

### Craft CMS version

5.0.5

### Plugin version

2.0.0-beta.1

### Multi-site?

No

### Additional context

**Composer.json** - (required)

```json
  "require": {
    "craftcms/ckeditor": "^4.0.3",
    "craftcms/cms": "^5.0.0",
    "craftcms/feed-me": "^6.0.0",
    "ether/seo": "^v5.0.0-rc1",
    "gustavs-gutmanis/craft-symfony-dump": "v2.0.0",
    "vlucas/phpdotenv": "^5.6.0",
    "wrav/oembed": "3.0.8"
  },
```

**Comment by engram-design** [comment]
I didn't actually think you needed this, but because we also require a beta version of `verbb/auth` that won't work without a `minimum-stability` directive.

If you add `"minimum-stability": "beta"` to your `composer.json` and try again, that should do the trick.

**Comment by engram-design** [comment]
Should be fixed in [2.0.0](https://github.com/verbb/social-share/releases/tag/2.0.0)
